google-cloud-clients/google-cloud-compute/src/main/java/com/google/cloud/compute/v1/stub/LicenseStubSettings.java

```diff
@@ -170,7 +170,7 @@ public LicenseStub createStub() throws IOException {
 
   /** Returns the default service endpoint. */
   public static String getDefaultEndpoint() {
-    return "https://compute.googleapis.com/compute/v1/projects/";
+    return "https://www.googleapis.com/compute/v1/projects/";
```

Comment:
@andreamlin Is this expected?

Reply:
https://www.googleapis.com/discovery/v1/apis/compute/v1/rest looks like they've changed it yeah.